volmaster/daemon.go

```diff
@@ -94,6 +82,7 @@ func (d *DaemonConfig) Daemon(debug bool, listen string) {
 	}
 
 	getRouter := map[string]func(http.ResponseWriter, *http.Request){
+		"/policy/{policy}":             d.handlePolicy,
```

Comment:
not related to this PR but looks like we have slight inconsistency between different GET endpoints, like for policy we have `policy/{policy}` but for volumes we have `/get/{policy}/{volume}` may be volume should also be accessible as `volume/{policy}/{volume}` ?